commit 29f5c43502910014257cd25a21a3c01ee4cf57b8
Author: Hui Wang <hui.wang@canonical.com>
Date:   Mon Feb 27 10:11:47 2017 +0800

    ALSA: hda - Fix micmute hotkey problem for a lenovo AIO machine
    
    commit 29693efcea0f38cf40d0055d2401490a4f9bf8be upstream.
    
    On this machine, the micmute button is connected to Line2 of the
    codec and the micmute led is connected to GPIO2 of the codec.
    
    After applying this quirk, both hotkey and led work well.
    
    Signed-off-by: Hui Wang <hui.wang@canonical.com>
    Signed-off-by: Takashi Iwai <tiwai@suse.de>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1238_29f5c43502910014257cd25a21a3c01ee4cf57b8.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1238_29f5c43502910014257cd25a21a3c01ee4cf57b8.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
